authpass/lib/ui/screens/entry_details.dart

```diff
@@ -1548,40 +1549,57 @@ class ObscuredEntryFieldEditor extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Stack(
+      alignment: Alignment.centerRight,
       children: [
-        InputDecorator(
-          decoration: InputDecoration(
-            prefixIcon:
-                commonField?.icon == null ? null : Icon(commonField.icon),
-            labelText: commonField?.displayName ?? fieldKey.key,
-            filled: true,
-          ),
-          child: const Text(
-            '*****************',
-            style: TextStyle(color: Colors.white),
-          ),
-        ),
-        Positioned.fill(
-          child: ClipRect(
-            child: BackdropFilter(
-              filter: ui.ImageFilter.blur(
-                sigmaX: 0.5,
-                sigmaY: 0.5,
+        Stack(
+          children: [
+            InputDecorator(
+              decoration: InputDecoration(
+                prefixIcon:
+                    commonField?.icon == null ? null : Icon(commonField.icon),
+                labelText: commonField?.displayName ?? fieldKey.key,
+                filled: true,
+              ),
+              child: const Text(
+                '*****************',
+                style: TextStyle(color: Colors.white),
               ),
-              child: LinkButton(
-                child: Container(
-                  alignment: Alignment.bottomCenter,
-                  padding: const EdgeInsets.only(left: 12.0 + 24.0, bottom: 16),
-                  child: const Text(
-                    'Protected field. Click to reveal.',
-                    style: TextStyle(
-                        shadows: [Shadow(color: Colors.white, blurRadius: 5)]),
+            ),
+            Positioned.fill(
+              child: ClipRect(
+                child: BackdropFilter(
+                  filter: ui.ImageFilter.blur(
+                    sigmaX: 0.5,
+                    sigmaY: 0.5,
+                  ),
+                  child: LinkButton(
+                    child: Container(
+                      alignment: Alignment.bottomCenter,
+                      padding: const EdgeInsets.only(left: 12.0 + 24.0, bottom: 16, right: 12),
+                      child: const Text(
+                        'Protected field. Click to reveal.',
+                        style: TextStyle(
+                            shadows: [Shadow(color: Colors.white, blurRadius: 5)]),
+                      ),
+                    ),
+                    onPressed: onPressed,
                   ),
                 ),
-                onPressed: onPressed,
               ),
             ),
-          ),
+          ],
+        ), 
+        IconButton(
+          icon: Icon(FontAwesomeIcons.eye),
+          color: ThemeUtil.iconColor(Theme.of(context), null),
+          tooltip: 'Show protected field',
+          onPressed: () {
+            FullScreenHud.show(context, (context) {
+              return FullScreenHud(
+                value: fieldKey.key,
```

Comment:
This shows the `key` of the field, not the actual value of the field. So people will always see "Password" 😅️

I think the best would be to add another `VoidCallback onShowPressed`  just like the `onPressed` parameter and let the parent widget show the Hud. It would also be nice if the two places (context menu and this button) are merged into one method of the parent widget (`_EntryFieldState`)

<img width="461" alt="Screen Shot 2020-09-29 at 13 26 43" src="https://user-images.githubusercontent.com/313066/94552604-6d0e5c00-0257-11eb-8ede-df339b8a39a9.png">